docs/source/notebooks/eccc-geoapi-xclim.ipynb

```diff
@@ -252,7 +252,7 @@
     "\n",
     "warnings.simplefilter(\"ignore\", category=numba.core.errors.NumbaDeprecationWarning)\n",
     "\n",
-    "import urllib\n",
+    "import urllib.request\n",
```

Comment:
The IDE did this?  Odd there is no accompany code change.

Reply:
I did this manually. There was an IDE warning being shown because `request` is not an imported module of `urllib` when taken from the top-level. All this does is get rid of a warning when users run the notebook manually.